advertising/transaction-message/reference.md

```diff
@@ -97,6 +97,9 @@ Defines the amenities that are part of a package deal.
 |Refundable|Required.<br /><br />An element that determines whether the room charge is refundable.<br /><br />**Attributes**<ul><li>available &mdash; Required. Set to true (or 1) if the room charge is refundable; otherwise, false (or 0).</li><li>refundable_until_days &mdash; Optional. The date by which the guest must request a refund. Specify the date as the number of days in advance of check-in. If not specified, the room charge is refundable up to and including the check-in date. Valid values are 0 through 180.</li><li>refundable_until_time &mdash; Optional. The guest may request a refund up until this time on the date referenced in the *refundable_until_days* attribute. Specify the time in the hotel's local time. Specify the time in the form, hh:mm:ss. For example, if the refund is available until 4 PM, use 16:00:00.</li></ul>|None
 |BreakfastIncluded|Optional.<br />Data type is Boolean.<br /><br />A value that determines whether the room bundle includes complimentary breakfast. Valid values are:<ul><li>1 (true)</li><li>0 (false)</li><li>true</li><li>false</li></ul> The default is false. You can use \<BreakfastIncluded/> as a shortcut for true.<br /><br />Don't include in a room bundle if the hotel provides free breakfast for all guests.|None
+|LunchIncluded|Optional.<br />Data type is Boolean.<br /><br />A value that determines whether the room bundle includes complimentary Lunch. Valid values are:<ul><li>1 (true)</li><li>0 (false)</li><li>true</li><li>false</li></ul> The default is false. You can use \<LunchIncluded/> as a shortcut for true.<br /><br />Don't include in a room bundle if the hotel provides free lunch for all guests.|None
+|DinnerIncluded|Optional.<br />Data type is Boolean.<br /><br />A value that determines whether the room bundle includes complimentary Dinner. Valid values are:<ul><li>1 (true)</li><li>0 (false)</li><li>true</li><li>false</li></ul> The default is false. You can use \<DinnerIncluded/> as a shortcut for true.<br /><br />Don't include in a room bundle if the hotel provides free dinner for all guests.|None
+|DrinksIncluded|Optional.<br />Data type is Boolean.<br /><br />A value that determines whether the room bundle includes complimentary drinks. Valid values are:<ul><li>1 (true)</li><li>0 (false)</li><li>true</li><li>false</li></ul> The default is false. You can use \<DrinksIncluded/> as a shortcut for true.<br /><br />Don't include in a room bundle if the hotel provides free drinks for all guests.|None
```

Comment:
Should we use the term Beverages for formal language? 
Also, do we need to add "Snacks Included"?